Week1/homework/app.js

```diff
@@ -1,11 +1,171 @@
+  changeMind: {
+    title: 'changeMind',
+    pages: 88,
+    author: 'author4',
+    released: 2008,
+    language: 'Arabic',
+  },
+  incendiaries: {
+    title: 'incendiaries',
+    pages: 65,
+    author: 'author5',
+    released: 2010,
+    language: 'English',
+  },
+  bloedFraude: {
+    title: 'bloedFraude',
+    pages: 188,
+    author: 'author6',
+    released: 2010,
+    language: 'Arabic',
+  },
+  milkMan: {
+    title: 'milkMan',
+    pages: 105,
+    author: 'author7',
+    released: 2017,
+    language: 'Dutch',
+  },
+  heavy: {
+    title: 'heavy',
+    pages: 88,
+    author: 'author8',
+    released: 2012,
+    language: 'Arabic',
+  },
+  theRecovering: {
+    title: 'theRecovering',
+    pages: 65,
+    author: 'author9',
+    released: 2018,
+    language: 'English',
+  },
+  cristinHannah: {
+    title: 'cristinHannah',
+    pages: 95,
+    author: 'author10',
+    released: 2019,
+    language: 'Arabic',
+  },
+};
 
-  // Replace with your own code
-  console.log(bookTitles);
+const bookSrc = {
+  lookAliveOut:
+    'http://t3.gstatic.com/images?q=tbn:ANd9GcSd9qk42ut0967WVXhQVa8exOndn0pHWj2EYTZ04WTcA274HGkf',
+  circe:
+    'http://t2.gstatic.com/images?q=tbn:ANd9GcQbzAjrxVH6ZiA_KczqrJuraqH0wNTEMt33G_Ppn-xLzc5iI-T7',
+  there:
+    'http://t0.gstatic.com/images?q=tbn:ANd9GcTb1YmRyJzI4UVmwdr4MRKDj6TWSKtMqTyyFoqOGYuITObg77To',
+  changeMind:
+    'http://t3.gstatic.com/images?q=tbn:ANd9GcRbvvRN_aCd_fJJjiINf9e1y7xW7FmXofxut8tI6Z-iXKrkwxsQ',
+  incendiaries:
+    'http://t0.gstatic.com/images?q=tbn:ANd9GcSgZYoEpcPleAOPQkHQgDmjvymuhpiJaMjUtD6IKLY6OcPKTBOK',
+  bloedFraude: 'https://s.s-bol.com/imgbase0/imagebase3/large/FC/2/7/9/2/9200000093712972.jpg',
+  milkMan:
+    'http://t0.gstatic.com/images?q=tbn:ANd9GcT-7v9nlkRW3sEoK3OnqqJ7Xt-3w2GfA4iJ-xeLOZY27XGs72ae',
+  heavy:
+    'http://t0.gstatic.com/images?q=tbn:ANd9GcQpEDMgFCei6EmdblXzSq3N2OzzXDwFI1QtEDx5PEe7vUctq1f8',
+  theRecovering:
+    'http://t0.gstatic.com/images?q=tbn:ANd9GcR1_RIZ37NjjiuqBcxr9b3BsA6-vPEoMDsbwKBtXYC5AdLd9qdd',
+  cristinHannah:
+    'http://t1.gstatic.com/images?q=tbn:ANd9GcQNeuab6cglxHh01N7hJ07keStQaSOcNaYIaIQumTwsFuH3whPq',
+};
+
+// ///////////////1.3   +  1.5/////////////////////////////
+function generate(obj) {
+  // extra code to create sound click
+  const sound = new Audio();
+  sound.src = 'sound/button-50.mp3';
+  function music() {
+    sound.play();
+  }
+  const ul = document.createElement('ul');
+  for (let x = 0; x < bookTitle.length; x++) {
+    const li = document.createElement('li');
+    const h3 = document.createElement('h3');
+    const p1 = document.createElement('p');
+    const p2 = document.createElement('p');
+    const p3 = document.createElement('p');
+    const p4 = document.createElement('p');
+    const img = document.createElement('img');
+    h3.onmouseover = music;
+    p1.onmouseover = music;
+    p2.onmouseover = music;
+    p3.onmouseover = music;
+    p4.onmouseover = music;
+    img.onclick = function() {
```

Comment:
The book cover according to the exercise should be added after the rest of the items had been created.  You were supposed to set the id attribute to the key, and then go and get the needed Dom element and change it.